packages/financial-templates-lib/src/price-feed/CreatePriceFeed.ts

```diff
@@ -587,11 +627,16 @@ function isMissingField(config, requiredFields, logger) {
   return false;
 }
 
-async function getUniswapPairDetails(web3, syntheticTokenAddress, collateralCurrencyAddress) {
+export async function getUniswapPairDetails(
+  web3: Web3,
+  syntheticTokenAddress: string,
+  collateralCurrencyAddress: string
+): Promise<{ address?: string; pairAddress?: string; inverted?: boolean }> {
   const networkId = await web3.eth.net.getId();
 
   if (process.env.UNISWAP_ADDRESS) {
     // Used for mock uniswap pair contracts.
+    // TODO: is address the wrong name for this field?
```

Comment:
hhmm. how should we track this todo? feels like it will just get lost in the repo once this is merged.

Reply:
We should just figure it out before merging. I'll use blame to find the original PR to determine if address is ever being used or if this field name was just incorrect.

Reply:
This is wrong. Will update the variable name.